packages/manager/modules/netapp/src/index.scss

```diff
@@ -0,0 +1,5 @@
+@import 'bootstrap4/scss/_functions.scss';
```

Comment:
Shouldn't it be scoped ? At least while there are cohabitation with BS3

Reply:
Yes, it should be scoped